backend/.env.example

```
@@ -0,0 +1,17 @@
+# Soroban RPC Configuration
+SOROBAN_RPC_URL=https://soroban-testnet.stellar.org:443
+SOROBAN_NETWORK_PASSPHRASE=Test SDF Network ; September 2015
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SOROBAN_NETWORK_PASSPHRASE key should go before the SOROBAN_RPC_URL key

(UnorderedKey)